MetaMask/metamask-mobile
# fix: connect request completed source validation

## **Description**

The `Connect Request Completed` analytics event had a wrong source `walletconnect` when a dapp was connecting to the wallet from the in-app browser. This PR fixes source the validation.

## **Related issues**

Fixes: [SDK-81](https://consensyssoftware.atlassian.net/browse/SDK-81)

## **Manual testing steps**

1. Connect from a dapp within the in-app browser
2. Observe Request Completed` analytics event being logged

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**



[SDK-81]: https://consensyssoftware.atlassian.net/browse/SDK-81?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
